routes/tasks.js

```diff
@@ -0,0 +1,10 @@
+
+const TasksController = require('../controllers/tasks');
+const router = require('express-promise-router')();
+
+router.route('/')
+    .get(TasksController.index)
+    .post(TasksController.newTask);
```

Comment:
add body validation with joi
don't forget to extract body from req.value.body in the controller function